src/System.Collections.Specialized/src/System.Collections.Specialized.builds

```diff
@@ -4,8 +4,14 @@
   <ItemGroup>
     <Project Include="System.Collections.Specialized.csproj" />
     <Project Include="System.Collections.Specialized.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="System.Collections.Specialized.csproj">
       <TargetGroup>net46</TargetGroup>
```

Comment:
We should be able to remove the already shipped builds and eliminate the complication of building them. So you shouldn't need the netstandard1.3 or net46 build configurations any longer.